libsolidity/formal/SMTEncoder.h

```diff
@@ -372,6 +372,12 @@ class SMTEncoder: public ASTConstVisitor
 	/// Returns true if _funDef was already visited.
 	bool visitedFunction(FunctionDefinition const* _funDef);
 
+	/// @returns FunctionDefinitions of the given contract (including constructors and inherited methods),
```

Comment:
The `including constructors` part is a bit misleading since the only constructor included is the one from contract.

Reply:
True, I have not changed the comment after the most recent change in the implementation.

Reply:
Can a contract have multiple constructors? Or at most one?

Reply:
At most one, OK.

Reply:
Updated the comment.